_includes/module-loading/bundling-webpack-es6-zip_editor.md

```diff
@@ -0,0 +1,50 @@
+1. Create a new source file for importing the required components from {{site.productname}} and configuring the editor.
+
+    {% include module-loading/bundling-required-components.md %}
+
+    Example `src/editor.js`
+
+    ```js
+    /* Import TinyMCE */
+    import '../tinymce/js/tinymce/tinymce';
```

Comment:
```suggestion
    import tinymce from '../tinymce/js/tinymce/tinymce';
```

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/2035/commits/f4202674bc44ca0b0ae13b1df30ba1a179bdc706